databricks_sql_connector/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "databricks-sql-connector"
+version = "3.3.0"
```

Comment:
As we change the package structure in this release. I would suggest to bump the major version to `4.0.0` or at least it should be greater than current release `3.4.0`,  so it has to start with `3.5.0`. @kravets-levko wdyt?